### ISSUE TITLE
Page footer bug since 1.1.0 version

### ISSUE THREAD

**Issue by Galixte** [david63/david63-cookiepolicy#6]
Hi,

i have a bug since 1.1.0 version. When i'm logged on footer page the Cookie acceptance box text is displayed without any Cookie acceptance box:
```
This forum uses cookies to give you the best and most relevant experience. By using this forum this means that you accept this policy.

You can find out more about the cookies used by clicking the "Cookie policy" on the main page.

[ I accept ]
```
If i'm not logged like guests, the Cookie acceptance box with the text are displayed correctly.

Three screenshots:
http://www.ezcom-fr.com/download/file.php?id=104
http://www.ezcom-fr.com/download/file.php?id=105
http://www.ezcom-fr.com/download/file.php?id=106

**Comment by david63** [comment]
This should be fixed in 1.1.1

**Comment by Galixte** [comment]
https://github.com/david63/david63-cookiepolicy/commit/9a06ec7d85501f27f02e06731ab4dba17c35b5a3

It's OK thanks ;)
